website/src/components/Header.tsx

```diff
@@ -126,7 +126,9 @@ export default class Header extends React.Component<HeaderProps, any> {
                                     )}
 
                                     <Nav.Link
-                                        className="btn btn-outline-primary ml-3 px-5 py-2"
+                                        className="btn btn-outline-primary ml-3 px-5 py-2 cta-btn"
```

Comment:
I think we should be cautious about using classes as a reference point for tracking since it may not be scalable long-term. If we ever make use of markup and style compression techniques, it may obfuscate our classes similarly how scss modules uses hashes. Also, if we're not styling anything with the `cta-btn` class, I think it would be more appropriate to add this data to a data-attribute since it may confuse future eyes thinking there is a `cta-btn` style somewhere. What do you think about adding `data-button-type="cta"` instead?

Reply:
I would be 100% fine with this, for all of the reasons you've mentioned. Tagging @adammaxla here, since I believe he made the initial recommendations on #5189. Adam, would it be cool on your end if we did it this way?